draggablepanel/res/values/attrs.xml

```diff
@@ -18,6 +20,8 @@
     <attr name="top_view_margin_right" format="dimension"/>
     <attr name="top_view_margin_bottom" format="dimension"/>
     <attr name="enable_minimized_horizontal_alpha_effect" format="boolean"/>
+    <attr name="enable_click_to_maximize" format="boolean"/>
+    <attr name="enable_click_to_minimize" format="boolean"/>
```

Comment:
This doesn't compile. You can't use the same "name" for attributes used with DraggablePanel and DraggableView. Remember to update the README.md with the new attr name.